### DISCUSSION TITLE
How to see LOCATION when use SQL Datawarehouse on Azure

### DISCUSSION THREAD

**Discussion by fernandodelago** [dbeaver/dbeaver#12635] [General]
Hi,

I have some external tables on Azure Datawarehouse, but when try to see the data LOCATION of the table, not was show to me on the DBeaver.
Only show the DDL of table (CREATE TABLE....field, field...;).
How can show more details about table when using DBeaver?
I use DBeaver 21.0.4.
Look this...

CREATE EXTERNAL TABLE [sellout].[EXT_TB_AJUSTES_AUT]
(
   [SALESORG_KEY] [varchar](250) NULL,
   [DISTR_CHAN_KEY] [varchar](250) NULL,
....
   [CONCATENADO1] [varchar](250) NULL
)
WITH (DATA_SOURCE = [ADL_GEN2_A4A_SOURCE_DEV]
,LOCATION = N'solutions/project/sellout/ajustes/gold_ajustes/fact_sellout_ajustes_aut'
,FILE_FORMAT = [FILE_FORMAT_PARQUET]
,REJECT_TYPE = VALUE
,REJECT_VALUE = 0)

In next, I try to see with VIEW TABLE (right-click of mouse):

CREATE TABLE [braz-corporate-sqldw].sellout.EXT_TB_AJUSTES_AUT_TEST (
	SALESORG_KEY varchar(10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
	DISTR_CHAN_KEY varchar(2) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
...
	ESTOQUE_VOLUME_KG varchar(255) COLLATE SQL_Latin1_General_CP1_CI_AS NULL
);

The code finish here.
Where is the information about LOCATION expecified on CREATE TABLE?

Thanks.

Fernando Delago

**Comment by LonwoLonwo** [comment]
Hello @fernandodelago 
This means that we have no additional information about location for the table to add it in in the table DDL.
You can create feature request for this here: https://github.com/dbeaver/dbeaver/issues
